rasa/core/policies/memoization.py

```diff
@@ -192,14 +193,20 @@ def _recall_states(self, states: List[State]) -> Optional[Text]:
         return self.lookup.get(self._create_feature_key(states))
 
     def recall(
-        self, states: List[State], tracker: DialogueStateTracker, domain: Domain,
+        self,
+        states: List[State],
+        tracker: DialogueStateTracker,
+        domain: Domain,
+        rule_only_data: Optional[Dict[Text, Any]],
     ) -> Optional[Text]:
         """Finds the action based on the given states.
 
         Args:
             states: List of states.
             tracker: The tracker.
             domain: The Domain.
+            rule_only_data: Slots and loops which are specific to rules and hence
+                should be excluded from during the featurization.
```

Comment:
```suggestion
            rule_only_data: Slots and loops which are specific to rules and hence
                should be ignored by this policy.
```
because we use "featurization" for too many things (e.g. in this case the "featurization" will not create numeric `Features` but just `State`s)